code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -402,9 +402,7 @@
 	return ..()
 
 /obj/item/melee/blood_magic/customised_abstract_text()
```

Comment:
Could you make the owner an argument for this function, and call that all of them as `abstract_item.customised_abstract_text(src)`?
```suggestion
/obj/item/melee/blood_magic/customised_abstract_text(mob/living/carbon/owner)
```

Reply:
done!